src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -628,7 +643,7 @@ void shouldFailWithParty_Details_InValid_Org_Individual_details_Present() {
             partyDetails.get(0).setOrganisationDetails(TestingUtil.organisationDetails());
             hearingRequest.setPartyDetails(partyDetails);
             Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
-                 .saveHearingRequest(hearingRequest,  null));
+                  .saveHearingRequest(hearingRequest, null,CLIENT_S2S_TOKEN));
```

Comment:
Not sure about the new leading space. (It's on a few lines).

Reply:
addressed the leading space